src/main/java/org/myrobotlab/service/data/Script.java

```diff
@@ -1,58 +1,21 @@
 package org.myrobotlab.service.data;
 
-import java.io.File;
 import java.io.Serializable;
 
 public class Script implements Serializable {
   static final long serialVersionUID = 1L;
   /**
    * unique location &amp; key of the script e.g. /mrl/scripts/myScript.py
    */
-  File file;
+  public String file;

```

Comment:
Should probably have getters, and if mutability is not needed then this and `code` should be final with no setters

Reply:
It can't be final, and its a bit of a philosophical and experience difference. 
To me POJOs are extremely dumb and simple data contracts, for some reason Java culture thinks setters and getters are always necessary.  They are not. Especially if the understanding of was a POJO is agreed on.  Other languages do not unnecessarily muddle thier POXOs with accessor functions..  POPOs, JavaScript Dictionaries, POCOs etc...
It is an enforced contract in the form of a object in memory.  Additionally, I've run into parsers that have had issues with accessor nuances